### PR TITLE
Remove unused variables

### PR DESCRIPTION
**Pull request by maratori**

### CODE REVIEW

File: lona/routing.py [hunk 1]
```diff
@@ -179,15 +179,11 @@ def _reverse(self, name, *args, **kwargs):
...
-        key_error = None
-
         try:
             return route.format_string.format(*args, **kwargs)
 
         except KeyError as e:
-            key_error = e
-
-        raise ValueError('missing URL arg: {} '.format(key_error.args[0]))
+            raise ValueError('missing URL arg: {} '.format(e.args[0]))
```
Comment (fscherf) [inline lona/routing.py:186]: This is actually on purpose to avoid stacking tracebacks

**Review by fscherf — CHANGES_REQUESTED** [review]
Great! I am ok with all changes except for one.

File: lona/routing.py (hunk 1 above)
Comment (maratori) [reply lona/routing.py]: Reverted and added comment

**Comment by fscherf** [comment]
Thanks!

For the future: It would be nice if you could split up these kinds fixes in smaller commits with commit messages for better orientation like ``imports: acquire: remove unused variable``.

**Pull request merged**